app/controllers/contesters_controller.rb

```diff
@@ -34,9 +37,20 @@ def create
 
   def update
     raise AccessError unless @contester.can_update? cuser
+
+    if @contester.contest.contest_type == Contest::TYPE_LADDER
+      old_rank = @contester.score
+      new_rank = params[:contester][:score].to_i
+      raise Error, t(:rank_invalid) unless new_rank > 0 and
+        new_rank <= @contester.contest.contesters.active.count
```

Comment:
Align the operands of a condition in an `unless` statement spanning multiple lines.